packages/components/package.json

```diff
@@ -28,6 +15,16 @@
   "files": [
     "dist"
   ],
+  "dependencies": {
+    "html-truncate": "^1.2.2",
+    "mime": "^2.5.2",
+    "react-intl": "^3.12.1",
+    "react-linkify": "^0.2.2",
+    "react-select": "^5.1.0",
+    "react-slick": "^0.24.0",
+    "react-text-format": "^2.0.28",
+    "snarkdown": "^2.0.0"
```

Comment:
So these are both dependencies and peer dependencies?

Reply:
Exactly, my end goal here is to make sure that we have access to those libraries with versions that are compatible with our code. But at the same time, if you have, let's say, `snarkdown@2.9.9` installed and it's compatible, we'll use this version instead of downloading the one specified in the dependency section.

Peer dependencies is a way of saying we support all those versions for this particular library. If you also have a supported version installed, we'll use it otherwise, install this version that comes from our dependency list.